src/cobra/core/compartment.py

```diff
@@ -4,11 +4,11 @@
 from warnings import warn
 
 from .dictlist import DictList
-from .object import Object
-from .. import Metabolite, Gene, Reaction, Model
+from .group import Group
+from .. import Metabolite, Reaction, Model
 
 
-class Compartment(Object):
+class Compartment(Group):
```

Comment:
I don't know if inheriting from `Group` is really desirable or we should just have a similar class interface.

Reply:
Another point in favor is that `.members` is a bit awkward here. We rather want attributes `.reactions` and `.metabolites`.